src/dialogs/alwaysonbotDialogs/UpdateProfile/UpdateAddress/getAddressesStep.ts

```diff
@@ -0,0 +1,315 @@
+import { LuisRecognizer } from "botbuilder-ai";
+import { Choice, ChoiceFactory, ChoicePrompt, ComponentDialog, DialogTurnResult, ListStyle, PromptValidatorContext, TextPrompt, WaterfallDialog, WaterfallStepContext } from "botbuilder-dialogs";
+import { CommonPromptValidatorModel } from "../../../../models/commonPromptValidatorModel";
+import { LUISAOSetup } from "../../../../utils/luisAppSetup";
+import i18n from "../../../locales/i18nconfig";
+import { ContinueAndFeedbackStep, CONTINUE_AND_FEEDBACK_STEP } from "../../Common/continueAndFeedbackStep";
+import { AddressDetails } from "./addressDetails";
+import { CallBackStep, CALL_BACK_STEP } from "./callBackStep";
+import { ChoiceCheckUpdateAddressStep, CHOICE_CHECK_UPDATE_ADDRESS_STEP } from "./choiceCheckUpdateAddressStep";
+import { UpdateAddressStep, UPDATE_ADDRESS_STEP } from "./updateAddressStep";
+
+const CHOICE_PROMPT = "CHOICE_PROMPT";
+const TEXT_PROMPT = "TEXT_PROMPT";
+
+let fullAddress: string;
+const MAX_ERROR_COUNT = 4;
```

Comment:
I'd like to see Master Error configured once in the app and reused, instead of repeated in every dialog.